src/console/Startup.cs

```diff
@@ -125,7 +126,30 @@ public virtual IEventConsumerService ResolveEventConsumerService(IServiceProvide
         {
```

Comment:
With the current changes I have in workspace-platform, this won't actually fix the issue. The workspace platform changes moves away from calling these "Resolve*" methods and instead just adds objects to the ServiceProvider, so it can handle the resolution.

We can definitely just add code there to to fix it, but it's important that we remember to do it, or the bug will still be there in Prod.

Reply:
Moved retry logic into Normalization Processor and reverted this back to its original form.